tasklist/client/src/common/i18n/locales/fr.json

```diff
@@ -33,6 +33,9 @@
     "loginPasswordFieldPlaceholder": "Mot de passe",
     "loginLoggingInMessage": "Connexion en cours",
     "loginButtonLabel": "Connexion",
+    "forbiddenPageTitle": "No tienes acceso a este componente",
+    "forbiddenPageDesc": "Parece que no tienes los permisos necesarios para acceder a este componente. <strong>Ponte en contacto con tu administrador de clúster</strong> para solicitar acceso.",
+    "forbiddenPageLinkLabel": "Más información sobre roles y permisos",
```

Comment:
@OmranAbazid You pasted the Spanish translations instead of the French ones here